lua/starfall/libs_sh/hologram.lua

```diff
@@ -298,7 +298,7 @@ else
 
 	--- Manually draws a hologram, requires a 3d render context
 	-- @client
-	-- @param bool? noTint If true, renders the hologram with it's color and opacity. This is the default behavior.
+	-- @param bool? noTint If true, renders the hologram with its color and opacity. The default is for holograms to render with no color or opacity, so use this argument if you need that behavior.
```

Comment:
`If true` still needs to be `If false or nil`

Reply:
Either that or change it to `If true, renders with hologram without its color or alpha`